cerebrium/endpoints/async.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: "Asynchronous Execution (Preview)"
+description: "Execute calls to a Cerebrium app to be run asynchroously"
+---
+
+<Note>
+This is a new feature! As such, the API is still currently subject to changes.
+
+Most applications are expected to work with the current implementation.
+However, should you encounter an issue deploying a Custom Runtime please
+reach out to us on Discord!
+
+</Note>
+
+Many applications require asynchronous execution of functions in a "fire-and-forget" fashion
+in order to work optimally. In this scenario, you hand off execution of a function to be Cerebrium's
+responsibility, while you as the developer are responsible for ensuring that data is able leave the function.
+
+Through parameterizing your request to the Cerebrium API, you can enable your function to executed asynchronously
```

Comment:
parameterizing your request sounds weird. Maybe say, by adding a query parameter to your endpoint url